sklearn/manifold/_mds.py

```diff
@@ -335,17 +347,24 @@ def smacof(
     >>> import numpy as np
     >>> from sklearn.manifold import smacof
     >>> from sklearn.metrics import euclidean_distances
-    >>> X = np.array([[0, 1, 2], [1, 0, 3],[2, 3, 0]])
+    >>> X = np.array([[0, 1, 2], [1, 0, 3], [2, 3, 0]])
     >>> dissimilarities = euclidean_distances(X)
-    >>> mds_result, stress = smacof(dissimilarities, n_components=2, random_state=42)
-    >>> np.round(mds_result, 5)
-    array([[ 0.05352, -1.07253],
-           [ 1.74231, -0.75675],
-           [-1.79583,  1.82928]])
-    >>> np.round(stress, 5).item()
-    0.00128
+    >>> Z, stress = smacof(dissimilarities, n_init=1, eps=1e-6, random_state=42)
+    >>> np.round(Z, 5)
+    array([[-0.03166,  1.10996],
+           [ 1.6829 ,  0.84601],
+           [-1.65124, -1.95597]])
```

Comment:
I think we don't really care about the actual numeric values of the component vectors in the doctest.

```suggestion
    >>> Z, stress = smacof(
    ...     dissimilarities, n_components=2, n_init=1, eps=1e-6, random_state=42
    ... )
    >>> Z.shape
    (3, 2)
```

Reply:
I agree. It was there before, so I kept it, but I agree that it's not needed.